sky/job/core.py

```diff
@@ -33,20 +33,21 @@ def launch(
     stream_logs: bool = True,
     detach_run: bool = False,
     retry_until_up: bool = False,
-) -> None:
+):
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
-    """Launch a managed spot job.
+    """Launch a managed job.
 
-    Please refer to the sky.cli.spot_launch for the document.
+    Please refer to the sky.cli.job_launch for the document.
```

Comment:
```suggestion
    Please refer to sky.cli.job_launch for documentation.
```
(document != documentation)

Reply:
Done. THanks!